revue-archeologique-du-centre-de-la-france.csl

```diff
@@ -277,7 +277,7 @@
         <choose>
           <if type="thesis">
             <text variable="genre" suffix=". "/>
-            <text macro="publisher" prefix=" "/>
+            <text macro="publisher"/>
```

Comment:
while I was looking at this style -- we should run a sed script to remove all empty affixes (i.e. prefix="" and suffix="") Since those never do anything, that can just be scripted and separate. @rmzelle might even consider including it in his general maintenance script?